packages/office-ui-fabric-react/src/components/GroupedList/GroupedList.styles.ts

```diff
@@ -25,11 +26,21 @@ export const getStyles = (props: IGroupedListStyleProps): IGroupedListStyles =>
         position: 'relative',
         fontSize: FontSizes.small,
         selectors: {
-          [`& :global(.${classNames.listCell})`]: {
-            minHeight: 36
+          [`.${classNames.listCell}`]: {
```

Comment:
we don't need :global anymore?

Reply:
I actually was not seeing the styles from this selector being applied at all, so I had to remove that for it to work.